src/logic/function/basic.lean

```diff
@@ -751,3 +751,7 @@ def set.separates_points {α β : Type*} (A : set (α → β)) : Prop :=
 
 lemma is_symm_op.flip_eq {α β} (op) [is_symm_op α β op] : flip op = op :=
 funext $ λ a, funext $ λ b, (is_symm_op.symm_op a b).symm
+
+lemma inv_image.equivalence {α : Sort u} {β : Sort v} (r : β → β → Prop) (f : α → β)
```

Comment:
It seems that `inv_image` is the same as `on_fun` (notation : `r on f`), and we have `equivalence.comap` for `on_fun`.

Reply:
Hmm. Why does both of them exist? Should I use `on_fun` instead?
Should there be a comment on one pointing to the other?

Reply:
Probably `on_fun` gives you a bit more API, if I understand @urkud correctly...

Reply:
It only appears in a proof here, so I suppose it doesn't matter for me anyway. 
But should we remove `inv_image` or at least marking it to be redundant in the docs since it seems be so?

Reply:
Both functions are in the core. Generally, we don't delete stuff from core unless we have a reason. We just ignore some parts.